src/ert/scheduler/job.py

```diff
@@ -187,6 +187,18 @@ async def _handle_failure(self) -> None:
             f"\n\t{self._callback_status_msg}"
         )
 
+        if (
+            self.iens in self.driver._job_error_message
+            and self.driver._job_error_message[self.iens]
+        ):
+            error_msg += (
+                f"\n\tDriver reported: {self.driver._job_error_message[self.iens]}"
+            )
```

Comment:
Is it not enough to only check if error message for specific iens exists?
```suggestion
        if (iens_error_msg := self.driver._job_error_message.get(self.iens)):
            error_msg += f"\n\tDriver reported: {iens_error_msg}"

```